src/data/nat/lattice.lean

```diff
@@ -64,6 +64,24 @@ begin
   rw nat.Inf_eq_zero, right, assumption,
 end
 
+lemma Inf_plus {n : ℕ} {p : ℕ → Prop} (h : 0 < Inf {m : ℕ | p m}) :
```

Comment:
```suggestion
lemma Inf_add {n : ℕ} {p : ℕ → Prop} (h : 0 < Inf {m : ℕ | p m}) :
```
Does this hold more generally though?

Reply:
For the record, it doesn't really. It requires the `Inf` to be realized and the default value to be `0`.